### PR TITLE
add_ableton-live-beta

### PR DESCRIPTION
**Pull request by yutarody**
#### Adding a new cask

- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] Checked the cask follows the requirements of [acceptable casks](https://github.com/caskroom/homebrew-versions#acceptable-casks).
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There are multiple errors. `audit` and `style` weren’t run.

**Comment by yutarody** [comment]
I'm sorry. I seem to have deleted the part that was modified when you merge commit rebase .

File: Casks/ableton-live-beta.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+cask 'ableton-live-beta' do
+  version '9.6.2b4'
+
+  if Hardware::CPU.is_32_bit?
+    sha256 '5d79a1e91efe0361559b6afaf6fefd4658e1f20625a50266e4a12798f62697c8'
+    url "http://cdn2-downloads.ableton.com/channels/#{version}/ableton_live_beta_#{version.no_dots}_32.dmg"
+    http://cdn2-downloads.ableton.com/channels/9.6.2b4/ableton_live_beta_962b4_32.dmg
```
Comment (vitorgalvao) [inline Casks/ableton-live-beta.rb]: This is still an error. This line should not be here.

Comment (yutarody) [inline Casks/ableton-live-beta.rb]: I'm sorry. I do not have yet a fix. It corresponds later.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in a08d32f90daf92fa708432351d505a7b3eed332d. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**